_includes/footer.html

```diff
@@ -1,21 +1,30 @@
 <footer class="footer grid">
-
   <div class="footer__wrapper grid-col-12">
-
-    <h2 class="footer__heading"><a class="brand" href="{{ site.baseurl }}/">OD<span class="brand__em">Ca</span></a></h2>
-
     <div class="footer__content grid--full">
-      <div class="footer__col grid-col-6">
-        <ul>
-          <li><a class="footer__nav-item" href="{{ site.baseurl }}/about/">About</a></li>
-          <li><a class="footer__nav-item" href="{{ site.baseurl }}/faq/">FAQ</a></li>
-          <li><a class="footer__nav-item" href="https://caciviclab.org/opendisclosure/">Join us</a></li>
-          <li><a class="footer__nav-item" href="https://github.com/caciviclab/odca-jekyll#license">Open source</a></li>
-        </ul>
+      <div class="footer__col footer__left-col grid-col-8">
+        <div class="footer__top-content">
+          <h2 class="footer__heading"><a class="brand" href="{{ site.baseurl }}/">Open<span class="brand__em">Disclosure</span><span class="brand__light"> Oakland</span></a></h2>
+          <ul class="footer__nav">
+            <li class="footer__nav-item"><a href="{{ site.baseurl }}/about/">About</a></li>
+            <li class="footer__nav-item"><a href="{{ site.baseurl }}/faq/">FAQ</a></li>
+            <li class="footer__nav-item"><a href="https://caciviclab.org/opendisclosure/">Join us</a></li>
+            <li class="footer__nav-item"><a href="https://github.com/caciviclab/odca-jekyll#license">Open source</a></li>
+            <li class="footer__nav-item"><a href="https://registertovote.ca.gov/">Register to vote</a></li>
+          </ul>
+        </div>
+        <div class="footer__bottom-content">
+          <h5 class="tagline">Brought to you by 
+            <span class="tagline__em"><a href="https://openoakland.org/">Open Oakland</a></span>
+            and
+            <span class="tagline__em"><a href="https://www.oaklandca.gov/boards-and-commissions/public-ethics-commission">Oakland's Public Ethics Commission</a></span>
+          </h5>
+          <h6 class="footer__stipulation">All contribution and expenditure data is provided directly by city officials for their respective city ballots</h6>
+        </div>
       </div>
-
-      <div class="footer__col grid-col-6">
-        <span class="tagline">Brought to you by <span class="tagline__em">California Civic Lab</span></span>
+      <div class="footer__col grid-col-4">
+        <div class="footer__voters-edge-widget">
+          <iframe title='Enter your address for personalized voting information' src='https://votersedge.org/en/address-widget' frameBorder='0' scrolling='no' style='width:240px;height:198px;'></iframe>
```

Comment:
I think the height could be bumped up to 200px. I'm seeing the bottom border get cut off.
![screenshot from 2018-09-09 22-45-28](https://user-images.githubusercontent.com/509703/45278515-428bb380-b482-11e8-8f00-cf495cbfedfb.png)